docs: architecture with the DevWorkspace engine (#2125)
codenvy/che-docs: modules/administration-guide/partials/con_postgresql.adoc

// Module included in the following assemblies:
//
// {prod-id-short}-workspace-controller-with-devworkspace

[id="postgresql_{context}"]
= PostgreSQL

> {prod-short} server uses the PostgreSQL database to persist user configurations: workspaces metadata, Git credentials.

Reviewer: A list of 2 items should have "and".
```suggestion
{prod-short} server uses the PostgreSQL database to persist user configurations: workspaces metadata and Git credentials.
```

The {prod-short} deployment starts a dedicated PostgreSQL instance in the `postgres` Deployment. 

> However, this deployment does not support full life-cycle capabilities, such as backups and recovery. For a business-critical setup, use an external database with the following recommended disaster-recovery options:

Reviewer: lifecycle - always one word as per the IBM SG

> However, this deployment does not support full life-cycle capabilities, such as backups and recovery. For a business-critical setup, use an external database with the following recommended disaster-recovery options:

Author: ```suggestion
However, this deployment does not support full lifecycle capabilities, such as backups and recovery. For a business-critical setup, use an external database with the following recommended disaster-recovery options:
```

* High Availability (HA)

> * Point In Time Recovery (PITR)

Reviewer: Perhaps, Point-In-Time Recovery? If the term isn't hyphenated throughout Che docs, please, ignore me :)

> * Point In Time Recovery (PITR)

Author: ```suggestion
* Point-In-Time Recovery (PITR)
```
